main.html

```diff
@@ -0,0 +1,1078 @@
+                                <div class="_2cuy _3dgx _2vxa"><span class="_4yxo">2. CP</span></div>
+                                <pre class="_19ik" dir="ltr"><span class="_2cuy _19ii _2vxa">A. <a href="https://l.facebook.com/l.php?u=https%3A%2F%2Fwww.codechef.com%2Fcertification%2Fprepare%3Ffbclid%3DIwAR1mqIri90FT97uxYmuledMFgfrzO_8EwtvP9He54Sj6dmS2GkQt0aE95x8&amp;h=AT0lmW-BolpFD0jY8FjPNFSiIdg5rHcdCSo_uAszwkhoQEjhiKIf9DRTyTX3R10lUq1EH8H7qIeFU_wxpni6-aG0XzMahW0AYT8dWg7SxepNoFo0Mq3x0xSkEctGXoD-CPXywg" target="_blank" rel="noopener nofollow" data-lynx-mode="asynclazy">https://www.codechef.com/certification/prepare</a> (9-10 topics)<br />B. <a href="https://l.facebook.com/l.php?u=https%3A%2F%2Fcodeforces.com%2Fblog%2Fentry%2F13529%3Ffbclid%3DIwAR3vhYV_Ue8sB62_PQSm3iQEsMbDoFdVLfL8eMLOVXz2s0wqCF4IWMCR5lc&amp;h=AT3yLPf7aRJ6FGLeEhA6sB-z7A-Mv9aDvSyit8EwOFqlFEqc_BbpLnWY9yn_1x9pKLaapdzZmz1mm56TLDn9aFp8FYNnPuj85ThuP-V5kLgXeBvFsZPKPWLHnS4t67a4bGCP1A" target="_blank" rel="noopener nofollow" data-lynx-mode="asynclazy">https://codeforces.com/blog/entry/13529</a></span></pre>
+                                <div class="_2cuy _3dgx _2vxa"><span class="_4yxo">----------------------------------------------------------------<br />----------------------------------------------------------------</span><br /><br /><span class="_4yxo">Terms Used-</span></div>
+                                <pre class="_19ik" dir="ltr"><span class="_2cuy _19ii _2vxa">CP: Competitive Programming</span></pre>
+                                <div class="_2cuy _3dgx _2vxa"><span class="_4yxo">Famous CP websites</span></div>
+                                <pre class="_19ik" dir="ltr"><span class="_2cuy _19ii _2vxa"><a href="https://l.facebook.com/l.php?u=https%3A%2F%2Fwww.codechef.com%2F%3Ffbclid%3DIwAR1d_Lh651dV369_PrvuVbRvhM4aTQa7610wOU1tebdrnd4WOo7GHCSxC10&amp;h=AT1etTOkmjtpK2Q7PYd9f4fqEoeyM541C57maAIovgp9s6koQ1cLY01MfGwO9gk483xBg4ClC3PAiXjsVylBj6Wk21vtcmZwbD5FrsD4HaDe1zpdINfb71PgCXeIWKzy66fTQw" target="_blank" rel="noopener nofollow" data-lynx-mode="asynclazy">codechef</a><br /><a href="https://l.facebook.com/l.php?u=http%3A%2F%2Fcodeforces.com%2F%3Ffbclid%3DIwAR1RsyFmpwuq-A4fbVXf_BzZ4jXH6CZHq7GoJEdvyTSR_a9W7wvV1NguCZY&amp;h=AT2TN8wtZDqYlppn3iwvu2Re0_2zHZoW08VYBHOOTLSKupjD_z_P_YYwov7UWkU8OkmMd2yWzpyafc4dKXrenIoSWFbTKiw_eI1TYGPsEhVFkYPkOqmY9-RIXNl97IqHx0C7kw" target="_blank" rel="noopener nofollow" data-lynx-mode="asynclazy">codeforces</a><br /><a href="https://l.facebook.com/l.php?u=https%3A%2F%2Fwww.hackerrank.com%2Fdashboard%3Ffbclid%3DIwAR3Ji1sNZ4ZpKoORg2_A18zkqEgKMIgYOOSMplBVPjR4LaHxhMIqur-lOlo&amp;h=AT2OzHRgtwt8a6SiB1h_FsR4MtpIa5G5oIxma7KLR9bp1Q8AGWhZ-0qijGDH
```

Comment:
Change the telegram link to `https://t.me/codeshows`

Reply:
ok i will make all the changes